packages/quantic/cypress/e2e/default-1/case-classification/case-classification-expectations.ts

```diff
@@ -170,6 +170,44 @@ function caseClassificationExpectations(selector: CaseClassificationSelector) {
         })
         .logDetail('should log the "ticket_classification_click" UA event');
     },
+
+    fetchClassificationsAfterValueChange: (
+      field: string,
+      predictions: Array<object>
+    ) => {
+      cy.wait(InterceptAliases.CaseClassification)
+        .then((interception) => {
+          const caseClassificationBody = interception.response?.body;
+
+          expect(caseClassificationBody.fields).to.haveOwnProperty(field);
+          expect(caseClassificationBody.fields[field]).to.haveOwnProperty(
+            'predictions'
+          );
+          expect(
+            caseClassificationBody.fields[field].predictions
+          ).to.deep.equal(predictions);
+        })
+        .logDetail(
+          'should fetch new case classifications after the value changes'
+        );
+    },
```

Comment:
I don't see really the point of testing the content of the response here. 
All what we need to test here is that there is indeed a new case classification request sent after selecting a new case classification. the content of the response doesn’t really matter.

It could just be something like this.
```suggestion
    fetchClassificationsAfterValueChange: () => {
      cy.wait(InterceptAliases.CaseClassification)
        .logDetail(
          'should fetch new case classifications after the value changes'
        );
    },
```

Reply:
Mmmm I thought there would be value in checking that we do get what we are expecting from the request, but I guess you are right in that it is not to us to test here.